.github/workflows/e2e-soak-trigger.yaml

```diff
@@ -0,0 +1,64 @@
+name: E2ESoakTrigger
```

Comment:
How frequently did we say that we are creating the clusters? I'd like to decouple the clusters creation from the cluster running in the case of soak testing. Ideally, we can just create a separate workflow that runs something like daily where we can run the `create-cluster` step